README.md

```diff
@@ -53,3 +53,12 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Testing
+
+To run the unit and integrations tests, run `poetry run pytest`.
+
+To run the end to end tests:
+- [Install Firefox](https://www.mozilla.org/en-GB/firefox/new/)
+- Download [geckodriver](https://github.com/mozilla/geckodriver/releases) (selecting the correct build for your OS / CPU architecture) and place it in the "bin" directory with the name "geckodriver"
```

Comment:
Minor, but you can leave the executable_path argument empty and it will look for a suitable executable on the PATH or in the project directory, which allows your developers to share their executable between multiple projects easily if they want to

Reply:
True, although in this case I didn't want to force changes to `PATH`. If developers want to share their executable from elsewhere then my expectation is that they'll place a symlink in the `bin` directory.